VM/rel2vm_params.py

```diff
@@ -343,22 +343,22 @@ def reduce_domain(
         # GMT spatial is not great-circle path connective
         geo.path_from_corners(
             corners=[ap, bp],
-            output="%s/tempEXT.tmp" % wd,
+            output=f"{wd}/tempEXT.tmp",
             min_edge_points=80,
             close=False,
         )
         isections, icomps = gmt.intersections(
-            ["%s/srf.path" % wd, NZ_LAND_OUTLINE.as_posix(), "%s/tempEXT.tmp" % wd],
+            [f"{wd}/srf.path", NZ_LAND_OUTLINE.as_posix(), f"{wd}/tempEXT.tmp"],
```

Comment:
Sorry for the back and forth but wd is actually a path so could just use the path operations rather than the f-string and then resolve it to a string for the function call.

Reply:
as you wish
```
 [(wd/"srf.path").as_posix(), NZ_LAND_OUTLINE.as_posix(), (wd/"tempEXT.tmp").as_posix()],
```